Resources/Markdown/package-collections.md

```diff
@@ -0,0 +1,39 @@
+---
+page-title: Package Collections
+description: The Swift Package Index supports the generation of package collections.
+---
+
+## Package Collections
+
+Package Collections are a new feature in the [Swift 5.5 release](XXX-LINK-TO-SWIFT-55-RELEASE-POST-XXX) of the Swift Package Manager allowing packages to be grouped, searched, and have their metadata inspected.
```

Comment:
I believe this link (and the one below which I'll mark in a second) are the only pending items. Maybe we should defer these so we can take the url public and update the links once Swift 5.5 and collections are released, avoiding the circular reference.